package.json

```diff
@@ -543,6 +543,7 @@
         "moment": "^2.27.0",
         "open": "^8.0.4",
         "semver": "^7.3.5",
+        "simple-git": "^3.5.0",
```

Comment:
Wouldn't the better approach be to update the `simple-git` dependency in `@microsoft/vscode-azext-azureappservice` and then update `@microsoft/vscode-azext-azureappservice` here?